# translation of src_debug_server.po to German
# Module src_debug_server translation file for language=de
# Copyright (C) 2003 Archaeopteryx Software, Inc.
# Wing Python IDE -- wingware.com
# Claudia Walenzyk <cwalenzyk@yahoo.com>, 2004.
#
msgid ""
msgstr ""
"Project-Id-Version: src_debug_server\n"
"POT-Creation-Date: 2020-05-13 12:03+EDT\n"
"PO-Revision-Date: 2016-12-19 20:56+0100\n"
"Last-Translator: Christoph Heitkamp <info@chrisheitkamp.de>\n"
"Language-Team: German\n"
"Language: de_DE\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"Generated-By: pygettext.py 1.4\n"
"X-Generator: Poedit 1.8.11\n"

#: src/debug/tserver/abstract.py:346
msgid "Wing Debug Log File\n"
msgstr "Wing Debug-Protokolldatei\n"

#: src/debug/tserver/abstract.py:347
msgid "This file can be used to diagnose problems in debugger internals\n"
msgstr ""
"Diese Datei kann genutzt werden, um Probleme in der Debugger-"
"Systemarchitektur festzustellen.\n"

#: src/debug/tserver/abstract.py:348
msgid "For help, contact support@wingware.com\n"
msgstr ""
"Wenn Sie Hilfe benötigen, kontaktieren Sie uns bitte über support@wingware."
"com.\n"

#: src/debug/tserver/abstract.py:353
msgid "Warning:  Log file does not exist:\n"
msgstr "Warnung: Protokolldatei existiert nicht:\n"

#: src/debug/tserver/abstract.py:355
msgid "Warning:  and could not be created; logging disabled\n"
msgstr ""
"Warnung: und konnte nicht erstellt werden; Protokollierung deaktiviert\n"

#: src/debug/tserver/abstract.py:358
msgid "Warning:  Log file is not a file:\n"
msgstr "Warnung: Protokolldatei ist keine Datei:\n"

#: src/debug/tserver/abstract.py:360
msgid "Warning:  Logging disabled\n"
msgstr "Warnung: Protokollierung deaktiviert\n"

#: src/debug/tserver/dbginspect.py:58 src/debug/tserver/dbgserver.py:3060
#: src/debug/tserver/dbgutils.py:489 src/debug/tserver/dbgutils.py:510
msgid "(truncated)"
msgstr ""

#: src/debug/tserver/dbginspect.py:929 src/debug/tserver/dbgutils.py:499
#: src/debug/tserver/dbgutils.py:520 src/debug/tserver/dbgutils.py:3156
msgid "<opaque "
msgstr "<unlesbar "

#: src/debug/tserver/dbgserver.py:1291
msgid "<module>"
msgstr "<Modul>"

#: src/debug/tserver/dbgserver.py:1433
msgid "<error evaluating>"
msgstr "<Fehler beim Bewerten>"

#: src/debug/tserver/dbgserver.py:2029 src/debug/tserver/dbgserver.py:2065
msgid "<thread %i is running>"
msgstr "<Thread %i läuft>"

#: src/debug/tserver/dbgserver.py:2032 src/debug/tserver/dbgserver.py:2068
msgid "<invalid thread id %s or stack index %s>"
msgstr "<ungültige Thread-id %s oder Stack-Index %s>"

#: src/debug/tserver/dbgserver.py:2034 src/debug/tserver/dbgserver.py:2070
msgid "<invalid stack frame>"
msgstr "<ungültiger Stack-Frame>"

#: src/debug/tserver/dbgserver.py:2098 src/debug/tserver/dbgserver.py:2127
msgid "<sys.modules does not exist>"
msgstr "<sys.modules existiert nicht>"

#: src/debug/tserver/dbgserver.py:2101 src/debug/tserver/dbgserver.py:2132
msgid "<error accessing sys.modules>"
msgstr "<Fehler beim Zugreifen auf sys.modules>"

#: src/debug/tserver/dbgserver.py:2129
msgid "<no longer exists"
msgstr "<existiert nicht mehr"

#: src/debug/tserver/dbgserver.py:2177 src/debug/tserver/dbgserver.py:2230
msgid "<no longer exists>"
msgstr "<existiert nicht mehr>"

#: src/debug/tserver/dbgserver.py:2208 src/debug/tserver/dbgserver.py:4518
msgid "<parent too large>"
msgstr "<parent zu lang>"

#: src/debug/tserver/dbgserver.py:2210 src/debug/tserver/dbgserver.py:4520
msgid "<invalid parent type>"
msgstr "<ungültiger übergeordneter Datentyp>"

#: src/debug/tserver/dbgserver.py:2220 src/debug/tserver/dbgserver.py:2689
msgid "<value not found>"
msgstr "<Wert nicht gefunden>"

#: src/debug/tserver/dbgserver.py:2677
msgid "<thread is running>"
msgstr "<Thread läuft>"

#: src/debug/tserver/dbgserver.py:2679
msgid "<invalid context>"
msgstr "<ungültiger Kontext>"

#: src/debug/tserver/dbgserver.py:4526
msgid "<invalid path %s at pos=%i>"
msgstr "<ungültiger Pfad %s an pos=%i>"

#: src/debug/tserver/dbgutils.py:151
msgid "<undefined>"
msgstr ""

#: src/debug/tserver/dbgutils.py:167
msgid "at"
msgstr ""

#: src/debug/tserver/dbgutils.py:177
msgid "len=%s>"
msgstr ""

#: src/debug/tserver/dbgutils.py:187 src/debug/tserver/dbgutils.py:2649
#, fuzzy
msgid "<error handling value>"
msgstr "<Fehler beim Bewerten>"

#: src/debug/tserver/dbgutils.py:189
msgid "<timeout>"
msgstr ""

#: src/debug/tserver/dbgutils.py:197
msgid "<huge "
msgstr ""

#: src/debug/tserver/dbgutils.py:217
msgid "<long "
msgstr ""

#: src/debug/tserver/dbgutils.py:234
#, fuzzy
msgid "<opaque"
msgstr "<unlesbar "

#: src/debug/tserver/dbgutils.py:512
msgid "<huge %s at %s; len=%i>"
msgstr "<huge %s an %s; Länge=%i>"

#: src/debug/tserver/dbgutils.py:726 src/debug/tserver/dbgutils.py:738
msgid ", pos "
msgstr ", pos "

#: src/debug/tserver/dbgutils.py:742
msgid ", line "
msgstr ", Zeile "

#: src/debug/tserver/dbgutils.py:744
msgid "SyntaxError: "
msgstr "SyntaxFehler:"

#: src/debug/tserver/dbgutils.py:772
msgid "Error"
msgstr "Fehler"

#: src/debug/tserver/dbgutils.py:782
msgid "Error: Wing was unable to pack up an exception message"
msgstr "Fehler: Wing konnte Exception-Nachricht nicht zusammenpacken"

#: src/debug/tserver/dbgutils.py:3518 src/debug/tserver/dbgutils.py:3534
msgid "Key"
msgstr ""

#: src/debug/tserver/dbgutils.py:3518 src/debug/tserver/dbgutils.py:3534
msgid "Value"
msgstr ""

#: src/debug/tserver/dbgutils.py:3529 src/debug/tserver/dbgutils.py:3556
msgid "Item"
msgstr ""

#: src/debug/tserver/netserver.py:472
msgid "Redirected I/O failed on closed debug connection"
msgstr "Umgeleiteter I/O an geschlossener Debug-Verbindung gescheitert"

#: src/debug/tserver/netserver.py:635
msgid "EOF when reading a line"
msgstr "Eingabe eines Dateiende-Zeichens (EOF) beim Lesen einer Zeile"

#: src/debug/tserver/netserver.py:1209
msgid "(unavailable)"
msgstr "(nicht verfügbar)"

#: src/debug/tserver/netserver.py:1472
msgid "[omitting some output]"
msgstr "[Ausgabe wird teilweise ausgelassen]"

#: src/debug/tserver/startdebug.py:205 src/debug/tserver/startdebug.py:390
#, fuzzy
msgid "wingdb.py: Error: Debug file does not exist:"
msgstr "Warnung: Protokolldatei existiert nicht:\n"

#: src/debug/tserver/startdebug.py:209 src/debug/tserver/startdebug.py:394
#, fuzzy
msgid "wingdb.py: Error: Debug file is not a file:"
msgstr "Warnung: Protokolldatei ist keine Datei:\n"

#: src/debug/tserver/startdebug.py:443
msgid "Network peer is "
msgstr ""

#: src/debug/tserver/startdebug.py:444
msgid "Attach port = "
msgstr ""

#: src/debug/tserver/startdebug.py:445
msgid "Initial stop = "
msgstr ""

#: src/debug/tserver/startdebug.py:459
msgid "wingdb.py: Could not create debug server"
msgstr ""

#: src/debug/tserver/startdebug.py:481
msgid "wingdb.py: Running %s"
msgstr ""

#: src/debug/tserver/startdebug.py:492 src/debug/tserver/startdebug.py:519
msgid "wingdb.py: Server exiting abnormally on exception"
msgstr ""

#: src/debug/tserver/startdebug.py:681
msgid "wingdb.py: Error: Failed to start the debug server"
msgstr ""

#: src/debug/tserver/startdebug.py:682
msgid "wingdb.py: Error: You may be running an unsupported version of Python"
msgstr ""

#: src/debug/tserver/startdebug.py:683
msgid "wingdb.py: Python version = %s"
msgstr ""

#: src/debug/tserver/startdebug.py:768 src/debug/tserver/startdebug.py:770
msgid "-- Type return or enter to exit --\n"
msgstr ""

#~ msgid "<unknown type>"
#~ msgstr "<unbekannter Typ>"

#~ msgid "Value too large to evaluate: "
#~ msgstr "Wert zu groß für Bewertung: "

#~ msgid "<invalid path registration>"
#~ msgstr "<ungültige Pfadregistrierung>"

#~ msgid "Function"
#~ msgstr "Funktion"

#~ msgid "Method"
#~ msgstr "Methode"

#~ msgid "Could not execute because an error occurred:\n"
#~ msgstr ""
#~ "Programm konnte nicht ausgeführt werden, weil ein Fehler aufgetreten "
#~ "ist:\n"

#~ msgid "Error: Runtime exception occurred"
#~ msgstr "Fehler: Laufzeit-Exception aufgetreten"
